### PR TITLE
[FE][CPF-75] My space ladder bucket

### PR DESCRIPTION
**Pull request by wiktoriasalamon**
image
image

### CODE REVIEW

File: frontend/src/components/modules/LevelDots/LevelDots.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { AdvancementLevelProps } from './AdvancementLevel.interface';
+import { LevelDotsProps } from './LevelDots.interface';
 import { MAX_LEVEL, MIN_LEVEL } from './constants';
 
-export const AdvancementLevel = ({ level }: AdvancementLevelProps) => {
+export const LevelDots = ({ level }: LevelDotsProps) => {
```
Comment (r1skz3ro) [inline frontend/src/components/modules/LevelDots/LevelDots.tsx:4]: we can replace it like we have in other components:
```suggestion
export const LevelDots: React.FC<LevelDotsProps> = ({ level }) => {
```

File: frontend/src/components/pages/mySpace/MySpaceBucketDetails/modules/AdvancementLevel/AdvancementLevel.interface.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { AdvancementLevel } from '@app/types/library';
+
+export interface AdvancementLevelProps {
+  data: AdvancementLevel;
+  showVerticalLine: boolean;
```
Comment (r1skz3ro) [inline frontend/src/components/pages/mySpace/MySpaceBucketDetails/modules/AdvancementLevel/AdvancementLevel.interface.ts:5]: what about just `verticalLine` 😄

Comment (wiktoriasalamon) [reply frontend/src/components/pages/mySpace/MySpaceBucketDetails/modules/AdvancementLevel/AdvancementLevel.interface.ts]: changed to `verticalLine`

File: frontend/src/components/common/ExpandableSection/ExpandableSection.interface.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export interface ExpandableSectionProps {
+  title: string;
+  description?: string;
+  showVerticalLine?: boolean;
```
[the remark this answers is not in the document]
Comment (wiktoriasalamon) [reply frontend/src/components/common/ExpandableSection/ExpandableSection.interface.ts]: changed to `verticalLine`

File: frontend/src/app/(app)/(root)/my-space/soft-skills/page.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        name: 'He is responsive and communicates responsibly - without unnecessary delay.',
+        description: loremIpsum,
+        proofStatus: ProofStatus.APPROVED as unknown as typeof ProofStatus,
+      },
+    ],
+    Quality: [
+      {
+        name: 'Fulfills undertaken obligations regarding tasks',
+        description: loremIpsum,
+        proofStatus: ProofStatus.APPROVED as unknown as typeof ProofStatus,
```
[the remark this answers is not in the document]
Comment (wiktoriasalamon) [reply frontend/src/app/(app)/(root)/my-space/soft-skills/page.tsx]: fixed

**Review by r1skz3ro — APPROVED** [review]

**Pull request merged**